patterns/java/src/refactoring_guru/patterns/template_method/network_template_method/Facebook.java

```diff
@@ -0,0 +1,22 @@
+package refactoring_guru.patterns.template_method.network_template_method;
+
+/**
+ * EN: Class of social network
+ *
+ * RU: Класс социальной сети.
+ */
+public class Facebook extends Network {
+    public boolean authenticate() {
+        return true;
```

Comment:
Add status messages and delays to each methods of this and other social network classes. Get inspiration from [Iterator pattern example](https://github.com/shvetsgroup/refactoring.guru-examples/blob/457ecef364463ccfacb91e2e8b76f64271b45a3f/patterns/java/src/refactoring_guru/patterns/iterator/example/social_networks/LinkedIn.java). It will make example output prettier + user won't wonder why these methods are empty.